julia/sr.jl

```diff
@@ -27,6 +27,10 @@ include("simplification.jl")
 
 include("PopMember.jl")
 
+
+include("halloffame.jl")
```

Comment:
Moved halloffame definition up to right after PopMember as this is the soonest it can be defined